feat: [Week-9] Translation HTML Builder (#2)
rolling-scopes-school/HTML-builder: README.md

# HTML builder

---

### Objectives:

The main goal of this series of tasks is to understand the basics of working with the **Node.js** platform.
You are offered to complete a series of small tasks with a gradual increase in complexity. The final touch will be the creation of a small application for building a static website.

Please note that most tasks can be accomplished in multiple ways, and at this stage, there are no incorrect approaches. Only the functionality of your code and adherence to the specified conditions will be evaluated.

---

## Execution process

- Fork this repository
- Send the repository link to `Cross-Check: Submit` in RS App
- Clone the repository
- Run the command `npm install` to install eslint, prettier and node.js typings that will assist you in completing the tasks
- Complete the tasks. The specifications for each task are in the `README.md` file inside the task folder
- Push the solution to your forked repository

## Crosscheck

> The process of cross-checking is described in the repository's wiki.  
> After starting the cross-check, follow [this link](https://github.com/rolling-scopes-school/HTML-builder/wiki) and follow the instructions on the page.

Reviewer: ```suggestion
The process for cross-checking is detailed in the repository's wiki.  
Once the cross-check begins, access the guidelines by following [this link](https://github.com/rolling-scopes-school/HTML-builder/wiki) and adhere to the instructions provided on the page.
```

### General Rules

> - The use of any third-party modules is prohibited.

Reviewer: ```suggestion
- The use of any third-party modules is strictly prohibited.
```

- Each task must be executed <u>in the root directory</u> using the command `node <task folder name>`.

> - The use of synchronous functions from the **fs module**, such as `fs.statSync(path[, options])`, `fs.readFileSync(path[, options])`, and others found in the [synchronous API section](https://nodejs.org/api/fs.html#fs_synchronous_api), is prohibited.

Reviewer: ```suggestion
- Utilizing synchronous functions from the **fs module**, such as `fs.statSync(path[, options])`, `fs.readFileSync(path[, options])`, and others found in the [Synchronous API section](https://nodejs.org/api/fs.html#fs_synchronous_api), is not allowed.
```

> - It is forbidden to use the `setTimeout()` timer.

Reviewer: ```suggestion
- The use of the `setTimeout()` function is forbidden.
```
